data_safe_haven/commands/deploy.py

```diff
@@ -28,6 +31,8 @@ def shm(
     """Deploy a Safe Haven Management component"""
     context = ContextSettings.from_file().assert_context()
     config = Config.from_remote(context)
```

Comment:
What if there's no remote file (e.g. on first run)?

Reply:
Fixed in 65ad597.

I've only put it in for deploy SHM for now (as the other commands couldn't work without this file already existing, I think).